### PR TITLE
Feat/homescreen/default

### PR DESCRIPTION
**Pull request by me-jterhorst**
I created my menu that slides in from the right.

### CODE REVIEW

File: client/src/index.css [hunk 1]
```diff
@@ -1,16 +1,54 @@
...
+  background: var(--black-85);
+}
+
+body {
+  border-radius: 25px;
+  background: var(--white-100);
+  border: 5px solid var(--black-100);
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
```
Comment (johnxgaehner) [inline client/src/index.css:53]: you did not deploy your app yet, so I can not see how your app looks but there is a problem for 100vh on mobile phones. I found a nice tool for that. here's an explanation of the problem and also the solution: https://www.npmjs.com/package/react-div-100vh

**Review by johnxgaehner — APPROVED** [review]
Hey Jakob,
looks super nice and well structured.
pls deploy your app so we can check it in the browser. :)))

File: client/src/components/BottomNav.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
+footer {
+  position: absolute;
+  inset: auto 0 0 0;
```
Comment (adrianammt) [inline client/src/components/BottomNav.css:3]: What does this do?

File: client/src/index.css [hunk 3]
```diff
@@ -6,11 +17,29 @@
...
+  font-weight: 400;
+  font-size: 16px;
+  letter-spacing: 2%;
+}
+
+body {
+  border-radius: 25px;
+  border: 5px solid var(--black-100);
+  display: flex;
+  min-height: 100vh;
```
Comment (mikemangas) [inline client/src/index.css]: im not sure, but couldnt u use something more relational than px?

File: client/src/components/BottomNav.css [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+.tabNav a.active {
+  position: relative;
+}
+
+.tabNav a.active::before {
+  content: "";
+  position: absolute;
+  bottom: -2px;
+  left: 50%;
+  transform: translateX(-75%);
```
Comment (adrianammt) [inline client/src/components/BottomNav.css:26]: I am sure this is a nice effect. Looking forward to seeing it deployed. ;)

**Review by ScienceMaster7 — APPROVED** [review]
Great Work 👍
keep going !

**Review by adrianammt — COMMENTED** [review]
It looks good to me!

File: client/src/components/BottomNav.css (hunk 2 above)
Comment (me-jterhorst) [reply client/src/components/BottomNav.css:3]: It's a shorthand, one-line-solution for: 

top: auto; 
right: 0; 
left: 0; 
bottom: 0;

**Review by namirsab — APPROVED** [review]
Hey @me-jterhorst good job,
Just a few comments, please take a look!

File: client/src/index.css [hunk 5]
```diff
@@ -1,16 +1,54 @@
...
-}
+@font-face {
+  font-family: "Open Sans";
+  src: local("Open Sans"), url(./fonts/OpenSans-Regular.ttf) format("truetype");
+}
+
+@font-face {
+  font-family: "Rubik";
+  src: local("Rubik"),
+    url(./fonts/Rubik-VariableFont_wght.ttf) format("truetype");
```
[the remark this answers is not in the document]
Comment (me-jterhorst) [reply client/src/index.css:9]: The fonts I use don't currently have a woff file available. 
I'd have to convert them later.

File: client/src/components/Header.js [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+              r='5.75'
+              className='icon-stroke--light'
+              stroke-width='2.5'
+            />
+            <path
+              d='M26.375 33.75L13.625 33.75C11.2088 33.75 9.25 31.7912 9.25 29.375C9.25 24.8877 12.8877 21.25 17.375 21.25L22.625 21.25C27.1123 21.25 30.75 24.8877 30.75 29.375C30.75 31.7912 28.7912 33.75 26.375 33.75Z'
+              className='icon-stroke--light'
+              stroke-width='2.5'
+            />
+          </svg>
```
[the remark this answers is not in the document]
Comment (me-jterhorst) [reply client/src/components/Header.js:138]: Awesome help guys, I implemented it already

File: client/src/index.css (hunk 1 above)
Comment (me-jterhorst) [reply client/src/index.css:53]: Thank you very much.

From what I gathered, It's not really my usecase as I am relying on min-height and not using exactly 100vh. But I'll investigate further

**Pull request merged**